src/integration.rs

```diff
@@ -0,0 +1,396 @@
+// Copyright © 2019 Intel Corporation
```

Comment:
2020, or just exclude the header (as we have the license file at the root of the project.